### PR TITLE
Add maintenance schedule check

### LINKED ISSUE #41: Plugin maint support
The maint plugin has the webeer tab. The schedule check is needed to use the maintenance schedules.  Proposed changes:  #40

### PR DESCRIPTION
**Pull request by unka65**
This adds the maint plugin schedule check.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ in GitHub.
 
 ## ChangeLog
 
+* feature:  Maintenance schedule check
```
Comment (netniV) [inline README.md]: Should have the associated issue number.

File: webseer_process.php [hunk 2]
```diff
@@ -98,6 +98,17 @@
 	exit(1);
 }
 
+if (api_plugin_is_enabled('maint')) {
+	include_once($config['base_path'] . '/plugins/maint/functions.php');
+}
+
+if (function_exists('plugin_maint_check_cacti_host')) {
```
Comment (netniV) [inline webseer_process.php]: Surely this should check for the correct function `plugin_maint_check_webseer_url` as it could be a version without it?

**Review by netniV — CHANGES_REQUESTED** [review]
Two minor changes and I think we'll be good to push it in. 👍

File: webseer_process.php (hunk 2 above)
Comment (unka65) [reply webseer_process.php]: I concur.  I incorrectly thought it was related to the API hook registration.

**Review by netniV — APPROVED** [review]

**Comment by netniV** [comment]
Thanks for your hard work, appreciated.

**Pull request merged**